apps/prairielearn/src/pages/instructorInstanceAdminSettings/instructorInstanceAdminSettings.tsx

```diff
@@ -244,7 +263,7 @@ router.post(
 
       const copiedInstance = await selectCourseInstanceByUuid({
         uuid: editor.uuid,
-        course: res.locals.course,
+        course: course as unknown as Course, // TODO: We need to write up proper model functions for Courses.
```

Comment:
Not blocking: Is it worth just doing this now? I don't mean to add a lot more work but this might be really easy.